google-cloud-spanner/src/main/java/com/google/cloud/spanner/DatabaseAdminClient.java

```diff
@@ -492,7 +495,7 @@ OperationFuture<Void, UpdateDatabaseDdlMetadata> updateDatabaseDdl(
   Operation getOperation(String name);
 
   /** Returns the IAM policy for the given database. */
-  Policy getDatabaseIAMPolicy(String instanceId, String databaseId);
+  Policy getDatabaseIAMPolicy(String instanceId, String databaseId, int version);
```

Comment:
The `version` argument needs some Javadoc. What does it do and when/why should I use it?